test_scripts/Resumption/InteriorVehicleData/003_InteriorVD_resumption_same_moduletype_with_different_moduleIds_unexpected_disconnect.lua

```diff
@@ -0,0 +1,82 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle subscription for same moduleType with different moduleIds data
+--  after transport disconnect
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to module_1 with modueId_1
+-- 4. App is subscribed to module_1 with modueId_2
+
+-- Sequence:
+-- 1. Transport disconnect and reconnect are performed
+-- 2. App starts registration with actual hashId after unexpected disconnect
+-- SDL does:
+-- - a. send RC.GetInteriorVD(subscribe=true, module_1, moduleId_1) and
+--     RC.GetInteriorVD(subscribe=true, module_1, moduleId_2) to HMI during resumption data
+-- - b. respond RAI(SUCCESS) to mobile app
+-- - c. update hashId after successful resumption
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/Resumption/InteriorVehicleData/commonResumptionsInteriorVD')
+
+--[[ Local Variables ]]
+local isSubscribed = true
+local moduleType = "SEAT"
+local moduleId1 = common.getModuleId(moduleType, 1)
+local moduleId2 = common.getModuleId(moduleType, 2)
+local appId = 1
+
+--[[ Local Functions ]]
+local function checkResumptionData()
+  local subscriptionsNumber  = 2
+  local actualModules = { }
+  local expectedModules = {
+    { moduleType = moduleType, moduleId = moduleId1 }, { moduleType = moduleType, moduleId = moduleId2 } }
+
+  common.getHMIConnection()("RC.GetInteriorVehicleData")
```

Comment:
```suggestion
  common.getHMIConnection():ExpectRequest("RC.GetInteriorVehicleData")
```
This comment relates to next scripts too.